client/js/components/common/feedback_icon.spec.js

```diff
@@ -0,0 +1,34 @@
+import React      from 'react';
```

Comment:
This shows what a good textbook-example this component is, not depending on anything about an assessment.